src/popup/router/components/FungibleTokens/TokensList.vue

```diff
@@ -52,13 +52,15 @@ export default {
      * Converts the token information object into a searchable list
      */
     convertedTokenInfo() {
-      return Object.entries(this.availableTokens).map(([contract, tokenData]) => ({
-        name: tokenData.name,
-        symbol: tokenData.symbol,
-        contract,
-        decimals: tokenData.decimals,
-        convertedBalance: tokenData.convertedBalance,
-      }));
+      return Object.entries(this.availableTokens)
+        .filter(([, tokenData]) => tokenData)
+        .map(([contract, tokenData]) => ({
+          name: tokenData.name,
+          symbol: tokenData.symbol,
+          contract,
+          decimals: tokenData.decimals,
+          convertedBalance: tokenData.convertedBalance,
+        }));
```

Comment:
in which case the value in [token info](https://raendom-backend.z52da5wt.xyz/tokenCache/tokenInfo) response can be `null`? Is it a backend issue? cc: @kenodressel 

if it is a case for some reason then proposing to filter it at [the top level](https://github.com/aeternity/superhero-wallet/blob/b03bde3e4ee32196de5a95c7bdcc20ba1d7c75f4/src/store/modules/fungibleTokens.js#L33-L35), right after data received to avoid breaking in other places for the same reason

Reply:
It can be null because we are fetching the token meta info from the contract via the node for every token. If a call fails it can be `null´.

Reply:
> in which case the value in [token info](https://raendom-backend.z52da5wt.xyz/tokenCache/tokenInfo) response can be `null`? Is it a backend issue? cc: @kenodressel
> 
> if it is a case for some reason then proposing to filter it at [the top level](https://github.com/aeternity/superhero-wallet/blob/b03bde3e4ee32196de5a95c7bdcc20ba1d7c75f4/src/store/modules/fungibleTokens.js#L33-L35), right after data received to avoid breaking in other places for the same reason

I agree that this should be filtered out on the top level, and yes the endpoint returns null for this contract `ct_cxp2TozAG6nzkQ84A1PyTWcKtgtPA52xTFkJPe2yxAeSdFnbY`

`ct_cxp2TozAG6nzkQ84A1PyTWcKtgtPA52xTFkJPe2yxAeSdFnbY":null,`

https://test-tipping.aeternity.art/tokenCache/tokenInfo